src/js/home/LayersPanel.jsx

```diff
@@ -1,22 +1,57 @@
 import React, { useEffect, useState } from "react";
 import { useAtom, useAtomValue } from "jotai";
+import styled from "@emotion/styled";
 
 import NICFIControl from "../components/NICFIControl";
 import ToolCard from "../components/ToolCard";
 
 // import i18n from "../i18n";
 import { extraMapParamsAtom } from "../home";
-import { startVisible, availableLayers } from "../constants";
+import { startVisible, availableLayers, layerColors } from "../constants";
 import { homeMapAtom } from "./HomeMap";
 import { useTranslation } from "react-i18next";
 
+const LayerCheckbox = styled.input`
+  accent-color: ${({ layerColor }) => layerColor || "var(--teal-1)"};
+`;
+
+const LayerSlider = styled.input`
+  background: ${({ layerColor }) =>
+    layerColor
+      ? `transparent linear-gradient(90deg, var(--gray-4) 0%, ${layerColor} 100%) 0% 0% no-repeat padding-box`
+      : "transparent linear-gradient(90deg, var(--gray-4) 0%, var(--black) 100%) 0% 0% no-repeat padding-box"};
+
+  &::-webkit-slider-thumb {
+    -webkit-appearance: none;
+    background: ${({ layerColor }) => layerColor || "var(--teal-1)"};
+    border: 2px solid;
+    border-color: var(--white);
+    border-radius: 50%;
+    box-shadow: 0px 3px 2px #00000040;
+    cursor: ew-resize;
+    height: 18px;
+    width: 18px;
+  }
+
```

Comment:
It also works when targeting both selectors with same css, instead of duplicating. Like `&::-webkit-slider-thumb,
  &::-moz-range-thumb {....}`